### PR TITLE
New package: tuxguitar-1.5.1

### PR DESCRIPTION
**Pull request by Johnnynator**
@abenson @cr6git

### CODE REVIEW

File: srcpkgs/tuxguitar/template [hunk 1]
```diff
@@ -0,0 +1,63 @@
+# Template file for 'tuxguitar'
+pkgname=tuxguitar
+version=1.5.1
+revision=1
+wrksrc="${pkgname}-${version}-src"
+hostmakedepends="apache-maven"
+makedepends="alsa-lib-devel fluidsynth-devel jack-devel"
+depends="virtual?java-runtime gtk+3-devel hicolor-icon-theme
```
Comment (abenson) [inline srcpkgs/tuxguitar/template]: `gtk+3-devel` in depends?

File: srcpkgs/tuxguitar/template [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+ desktop-file-utils"
+short_desc="Multitrack guitar tablature editor and player"
+maintainer="Andrew Benson <abenson+void@gmail.com>"
```
Comment (abenson) [inline srcpkgs/tuxguitar/template]: ...

File: srcpkgs/tuxguitar/template (hunk 1 above)
Comment (Johnnynator) [reply srcpkgs/tuxguitar/template]: wtf that should have been gtk+3

File: srcpkgs/tuxguitar/template (hunk 2 above)
Comment (Johnnynator) [reply srcpkgs/tuxguitar/template]: Mh... I just had copied you template. Can set the Maintainer to myself.

Comment (abenson) [reply srcpkgs/tuxguitar/template]: It doesn't matter

**Pull request merged**